nebula/ui/components/VPNServerCountry.qml

```diff
@@ -166,7 +166,10 @@ VPNClickableRow {
             delegate: VPNRadioDelegate {
                 property string _cityName: modelData[0]
                 property string _localizedCityName: modelData[1]
+                property string _activeServerCount: modelData[2]
                 property string _countryCode: code
+                property bool isAvailable: _activeServerCount > 0
```

Comment:
@oskirby I just would like to check if I understand what is happening in PR https://github.com/mozilla-mobile/mozilla-vpn-client/pull/2537 ([this part in particular](https://github.com/mozilla-mobile/mozilla-vpn-client/pull/2537/files#diff-4a9d949202ba311f8e12ee3251f43dd2cbe5e05405f18699628652fa20ece217R160-R172)) correctly: We can say an `activeServerCount` of zero means that all of the servers in a city are _timed out_ so the city is currently “unavailable”?

Reply:
That is correct. The `activeServerCount` is the number of servers that not been blocked due to a failed handshake. When the count reaches zero, there are no servers remaining that the VPN client can use for this location.